From c5ede25821f503d109e48be678e29a6acca13fa1 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 30 Sep 2020 01:39:42 -0400
Subject: [PATCH 025/148] Revert "FODAnimation: Check if view is null before
 adding a new one"

This reverts commit 1f24a8b24eb24be37c669e6c33db130ae4d2745e.
---
 .../src/com/android/systemui/biometrics/FODAnimation.java   | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/biometrics/FODAnimation.java b/packages/SystemUI/src/com/android/systemui/biometrics/FODAnimation.java
index d687ca0df21..43a08c447df 100644
--- a/packages/SystemUI/src/com/android/systemui/biometrics/FODAnimation.java
+++ b/packages/SystemUI/src/com/android/systemui/biometrics/FODAnimation.java
@@ -92,10 +92,8 @@ public class FODAnimation extends ImageView {
     public void showFODanimation() {
         if (mAnimParams != null && !mShowing && mIsKeyguard) {
             mShowing = true;
-            if (this.getWindowToken() == null) {
-                mWindowManager.addView(this, mAnimParams);
-                mWindowManager.updateViewLayout(this, mAnimParams);
-            }
+            mWindowManager.addView(this, mAnimParams);
+            mWindowManager.updateViewLayout(this, mAnimParams);
             recognizingAnim.start();
         }
     }
-- 
2.17.1

